swift/app.py

```diff
@@ -38,3 +40,31 @@ def frames(self) -> Iterable[QWidget]:
             An iterable object of Frame objects for showing.
         """
         return ()
+
+    def broadcast(self, channelName: str, content: Any):
+        """Broadcasts the content to the target channel.
+
+        Args:
+            channelName: Target channel name.
+            content: Content to be broadcast.
+        
+        """
+        try:
+            content = json.dumps(content)
+        except TypeError as e:
+            print(f"swift.app.broadcast(): {e!r}")
+            return
+        self.broadcastRequested.emit(channelName, content)
+
+    def receivedSlot(self, channelName: str, content: Any):
+        """This will be overridden by child classes."""
```

Comment:
Please describe more about this slot, for example:
```python
"""Handles the received broadcast messages.

This is called when self.received signal is emitted.
The child class may override this method to implement a specific handler.
"""

Reply:
Ok! Also, I added arguments descriptions.